jadx-gui/src/main/resources/i18n/Messages_de_DE.properties

```diff
@@ -40,9 +40,9 @@ tree.resources_title=Ressourcen
 tree.loading=Laden…
 
 progress.load=Laden
-#progress.export_mappings=
+progress.export_mappings=Zuordnungen exportieren
 progress.decompile=Dekompilieren
-#progress.canceling=Canceling
+progress.canceling=Abbrechen
```

Comment:
This entry is shown after user has chosen to cancel a task. So in my opinion a better translation would be `breche ab`. It is used in this way: `progressPane.changeLabel(this, task.getTitle() + " (" + NLS.str("progress.canceling") + ")… ");`

Reply:
Breche ab, brechen sie ab, Abbrechen

Reply:
@Henry2o1o In my opinion `Breche ab` would be best